The patch contributed to upstream that resolves yelp
or minibrowser crash during the program termination.
Please see the upstream bug for details:
https://bugs.webkit.org/show_bug.cgi?id=197266

Index: Source/WTF/wtf/generic/MainThreadGeneric.cpp
===================================================================
--- a/Source/WTF/wtf/generic/MainThreadGeneric.cpp	(revision 250087)
+++ b/Source/WTF/wtf/generic/MainThreadGeneric.cpp	(working copy)
@@ -3,6 +3,7 @@
  * Copyright (C) 2007 Justin Haygood (jhaygood@reaktix.com)
  * Copyright (C) 2016 Konstantin Tokavev <annulen@yandex.ru>
  * Copyright (C) 2016 Yusuke Suzuki <utatane.tea@gmail.com>
+ * Copyright (C) 2019 Oracle and/or its affiliates. All rights reserved.
  *
  * Redistribution and use in source and binary forms, with or without
  * modification, are permitted provided that the following conditions
@@ -36,6 +37,7 @@
 #endif
 
 #include <wtf/RunLoop.h>
+#include <wtf/NeverDestroyed.h>
 #if USE(GLIB)
 #include <wtf/glib/RunLoopSourcePriority.h>
 #endif
@@ -101,8 +103,8 @@ void scheduleDispatchFunctionsOnMainThre
 {
     // Use a RunLoop::Timer instead of RunLoop::dispatch() to be able to use a different priority and
     // avoid the double queue because dispatchOnMainThread also queues the functions.
-    static MainThreadDispatcher dispatcher;
-    dispatcher.schedule();
+    static NeverDestroyed<MainThreadDispatcher> dispatcher;
+    dispatcher.get().schedule();
 }
 
 } // namespace WTF
